Sources/Testing/Testing.docc/testing-for-errors-in-swift-code.md

```diff
@@ -48,8 +48,8 @@ running your test if the code doesn't throw the expected error.
 
 To check that the code under test throws an error of any type, pass
 `(any Error).self` as the first argument to either
-``expect(throws:_:sourceLocation:performing:)-1xr34`` or
```

Comment:
@heckj FYI fixed some typos here that I noticed when building the DocC bundle.

Reply:
With any luck these hashes can be replaced in 6.1+ (somewhere downstream) with annotations that are more directly related to signatures and types. It doesn't take much of a change to throw off the hashes into something new, which is really annoying (to me)

Reply:
That's a question for the DocC folks, at any rate.